### ISSUE TITLE
Syntax checking ignoring flycheck eslint? 

### ISSUE THREAD

**Issue by nocksock** [syl20bnr/spacemacs#10562]
In our project we use the eslint style standard and while eslint itself works fine, and `SPC e l` yields the correct linting errors, the syntax checking complains about missing `;`

What's going on, and how do I fix this? 

## Example:
![screen shot 2018-04-05 at 11 40 08](https://user-images.githubusercontent.com/22164/38358719-56eaf620-38c6-11e8-9c85-af24da3b1746.png)

## `SPC e v`:
![screen shot 2018-04-05 at 11 43 50](https://user-images.githubusercontent.com/22164/38358824-a805c792-38c6-11e8-8a6e-07677e481f0f.png)

**Comment by CeleritasCelery** [comment]
Run `flycheck-compile` to see the actual command line used to run eslint.

**Comment by nocksock** [comment]
it's the correct command. 

However I found out, that it's the the warnings of js2-mode, which uses `jshint` instead of eslint. I can toggle the jshint warnings/errors with `, w`.

Added this to my .spacemacs to `dotspacemacs/user-init ()`, to disable it by default:

    ;; Turn off js2 mode errors & warnings (we lean on eslint/standard)
    (setq js2-mode-show-parse-errors nil)
    (setq js2-mode-show-strict-warnings nil)

I wonder if this should be spacemacs-default, since flycheck is integrated so nicely and I've seen this lead to confusion.
